docker-compose.yml

```diff
@@ -123,6 +123,17 @@ services:
     env_file:
       - ./analytics/env.docker
 
+  proxy:
+    build: ./nginx
+    image: openverse-proxy
+    ports:
+      - "80:80"
+      - "443:443"
```

Comment:
Should these proxy on the ports we're already accustomed to finding the API on, rather than 80/443?

Reply:
This allows us to use the standard ports for HTTP and HTTPs will continuing to expose the Gunicorn service directly on 8000 (to debug or see logs).

Reply:
Binding low ports requires root access on Linux (not sure about other OSs). I run rootless docker so this doesn't even work for me with my typical workflow.

I could run just this stack as root, but that obviates the security benefits of using rootless docker :shrug:

Reply:
Rootless Docker is good. I hadn't considered that scenario. What ports other than 8000 would be okay, (considering all the occupied ports and 8443 being taken by frontend)?

Reply:
`wp-env` runs the HTTPS version one port higher than the HTTP version. We could do the same here and use 8001 (though preferably we wouldn't be relying on reserved port spaces and move to something in the [unreserved range](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Dynamic,_private_or_ephemeral_ports) :stuck_out_tongue: not necessary to change that for this PR though, it's something we would need to address over the whole stack).